bgpd/bgpd.c

```diff
@@ -8229,6 +8229,7 @@ void bgp_terminate(void)
 	/* reverse bgp_master_init */
 	for (ALL_LIST_ELEMENTS(bm->bgp, mnode, mnnode, bgp)) {
 		bgp_close_vrf_socket(bgp);
+		THREAD_OFF(bgp->t_condition_check);
```

Comment:
doesn't this belong in bgp_delete() around line 3618?

Reply:
Agree, moved there.